src/components/augmentedReality/AugmentedReality.js

```diff
@@ -50,18 +50,24 @@ export const AugmentedReality = ({
   const [isModalVisible, setIsModalVisible] = useState(false);
   const selectedFilterId = filter.find((entry) => entry.selected)?.id;
 
-  useEffect(() => {
-    const { settings = {} } = globalSettings;
-
+  const supportAr = async () => {
     try {
-      !!settings.ar &&
-        isARSupportedOnDevice(
-          () => null,
-          () => setIsARSupported(true)
-        );
+      const isARSupported = (await isARSupportedOnDevice())?.isARSupported;
+
+      if (isARSupported) {
+        setIsARSupported(true);
+      }
     } catch (error) {
       // if Viro is not integrated, we need to catch the error for `isARSupportedOnDevice of null`
-      console.warn(error);
+      console.error(error);
```

Comment:
why did you switch to `error` instead of `warn`? wouldn't this crash the app entirely?

Reply:
No, this is just a log type. If we need to show an error in the `catch` section, I prefer to show it via `error` instead of `warn`.